docs/build/abci++/00-introduction.md

```diff
@@ -0,0 +1,52 @@
+# Introduction
+
+## What is ABCI++?
+
+ABCI++, Application Blockchain Interface is the interface between CometBFT and the application. Within the release of ABCI 2.0 for the 0.38 CometBFT release there were additional methods introduced.
+
+The 5 methods introduced during ABCI 2.0 are:
+
+- `PrepareProposal`
+- `ProcessProposal`
+- `ExtendVote`
+- `VerifyVoteExtension`
+- `FinalizeBlock`
+
+
+## The Flow
+
+## PrepareProposal
+
+Based on their voting power, CometBFT chooses a block proposer and calls `PrepareProposal` on the block proposer's application (Cosmos SDK). The selected block proposer is responsible for collecting outstanding transactions from the mempool, adhering to the application's specifications. The application can enforce custom transaction ordering and incorporate additional transactions, potentially generated from vote extensions in the previous block.
+
+To perform this manipulation on the application side, a custom handler is implemented. If no custom handler exists, all transactions are validated. Please see [this](https://github.com/fatal-fruit/abci-workshop) tutorial for more information on custom handlers.
```

Comment:
will this tutorial get moved to tutorials? its a bit weird we link to a random repo here. Could be confusing for users

Reply:
yep, thats the plan for this week but needed to get something out asap for people to use. Its only for the interim. Following moving the tutorial across this link will point to the `sdk-tutorials` repo. I mentioned here that the repo will be moved across https://github.com/cosmos/cosmos-sdk-docs/pull/91 but should of made mention here too.